Tests/SwiftProtobufTests/Test_Required.swift

```diff
@@ -167,7 +167,7 @@ class Test_Required: XCTestCase, PBTestHelpers {
     }
```

Comment:
Just noticed this:  We should have a real text format serializer method in addition to the `debugDescription` hook (because people actually do use text format as an interchange format).  That would remove the need to play games just above to massage the expected text into the `debugDescription` format.